code/modules/fulton/fulton.dm

```diff
@@ -0,0 +1,163 @@
+var/list/normal_extraction_beacons = list()
+var/list/total_extraction_beacons = list()
+var/list/extraction_appends = list("-AAAAAAAAAAAAAAAAAUGH", "-AAAAAAAAAAAHHHHHHHHHH")
+
+/obj/item/weapon/extraction_pack
+	name = "fulton recovery pack"
+	desc = "A balloon that can be used to extract a target to a Fulton Recovery Beacon. Anything not bolted down can be moved. Link the pack to a beacon by using the pack in hand."
+	icon = 'icons/obj/fulton.dmi'
+	icon_state = "extraction_pack"
+	var/obj/machinery/extraction_point/beacon
+	var/is_syndie = 0
+	var/uses_left = 12
+
+/obj/item/weapon/extraction_pack/examine(mob/user)
+	. = ..()
+	user.show_message("It has [uses_left] uses remaining.", 1)
+
+/obj/item/weapon/extraction_pack/advanced
+	name = "advanced fulton recovery pack"
+	uses_left = 24
+
+/obj/item/weapon/extraction_pack/super
+	name = "super fulton recovery pack"
+	uses_left = 48
+
+/obj/item/weapon/extraction_pack/syndicate
+	name = "syndicate fulton recovery pack"
+	is_syndie = 1
+	uses_left = 9999
+
+/obj/item/weapon/extraction_pack/attack_self(mob/user)
+	if(!total_extraction_beacons.len)
+		user << "There are no extraction beacons in existence!"
+		return
+	else
+		var/A
+		if(is_syndie)
+			A = input("Select a beacon to connect to", "Fulton Recovery Pack", A) in total_extraction_beacons
+		else
+			A = input("Select a beacon to connect to", "Fulton Recovery Pack", A) in normal_extraction_beacons
```

Comment:
Replace this with a car pointing to which list it picks the beacon from. Kill `is_syndie`.